### ISSUE TITLE
p-inputNumber Accessibility Issues

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#326]
### Describe the bug

The p-inputNumber component has two accessibility violation when running the IBM Equal Accessibility Checker: 
1. The ARIA attribute "aria-valuemin" is in conflict with the corresponding HTML attribute "min"
2. The ARIA attribute "aria-valuemax" is in conflict with the corresponding HTML attribute "max"

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://tcthglgb.stackblitz.io 

### Environment

Stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the errors: 'The ARIA attribute "aria-valuemin" is in conflict with the corresponding HTML attribute "min"' and 'The ARIA attribute "aria-valuemax" is in conflict with the corresponding HTML attribute "max"'

### Expected behavior

No accessibility violations are present for the p-inputNumber component

> Migrated from `primefaces/primeng#18814` — originally by `@aciesla4` on 2025-08-28

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2026-02-25T00:29:29.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!
